geopandas/array.py

```diff
@@ -891,6 +891,41 @@ def unique(self):
     def nbytes(self):
         return self.data.nbytes
 
+    def shift(self, periods: int = 1, fill_value: object = None) -> "GeometryArray":
```

Comment:
```suggestion
    def shift(self, periods=1, fill_value=None):
```

We currently do not use function annotations, so the suggestion for the sake of consistency.